docs/topic-guides/blendable-web-apps/customizing-solutions-design.md

```diff
@@ -0,0 +1,92 @@
+
+Naturally, you can create your own, and ship it with an app.
+
+
+## Collections of CSS Custom Properties
+
+Slightly lower-level way to customize existing design is by using CSS Variables (Custom Properties). Well-behaved design systems, themes, custom elements and even individual CSS rules usually use CSS Variables which could be used to customize it.
+
+This allows to change color schemes, or configure other values like font sizing, margins, etc.
+
+Again [Blending](https://github.com/Starcounter/Blending) apps suite [delivers GUI](https://github.com/Starcounter/Blending/blob/master/docs/gui.md#css-variables-manager) to do that without touching any code. You can as well add it to you composition code manually.
+
+{% hint style="warning" %} Attaching CSS Variables' collection through Blending app, allows you to set them on light DOM. Currently to the global scope - `body` (https://github.com/Starcounter/Blending/issues/232). While setting it in view's composition, attaches them in Shadow DOM to the scope you choose. {% endhint %}
+
+## Overwriting `uni-elements` definitions
+
+That is the lowest-level approach, that is slightly hacky, but this hackiness allows you to do any kind of changes and re-design.
+
+It allows you to completely change not only look but also the behavior of any UI element.
+
+Assuming, the solution of apps is created consistently using Starcounter Design System and Uniform Components, for example, all date pickers are presented using `<uni-date-picker>`.
+
+Then if you want to drastically change its look or even behavior, you can do so, by creating your own custom element with the same name, but with more sophisticated behavior.
+
+For example, an airline company could have high demands on the UX and feature set of the date picker. With Starcounter, it could still use an app which was created with regular date picker needs.
+
+The requirements are that initial design author has stuck to the conventions of Starcounter Design System, and the new date picker supports the same API, which in case of Uniform Components is just nothing more than being able to wrap the native HTML elements. That means no specific API, just exchange data with native HTML API.
+
+You can fork [`uni-element`](https://github.com/Starcounter/uniform.css/tree/master/components) or start from scratch.
+
+Then you need to add this definition to any apps `/sys` folder under the same path as an overwritten element, for example `/sys/uniform.css/components/uni-date-picker/uni-date-picker.html`. You can use a fork of  [StarcounterClientFiles app](https://github.com/Starcounter/StarcounterClientFiles/) for this purpose.
+
+
+{% hint style="info" %} The same technique works for any custom element, not only Uniform one. {% endhint %}
+
+## Need more?
+
+If any of the above fulfill your needs for general customization and theming, first of all, please let us know.
```

Comment:
Would it make sense to give people a way to let us know?

Reply:
Definitely the question is what is this way, GH issue, SO question?

Reply:
Exactly, since it doesn't fit into a GitHub issue, I'm not sure. If we don't have a clear way for people to contact us about this, I suggest that we remove this sentence.